crates/compiler/gen_dev/src/generic64/mod.rs

```diff
@@ -2791,6 +2907,10 @@ impl<
                     tags,
                 );
             }
+            UnionLayout::NonNullableUnwrapped(_) => {
```

Comment:
If it is non-nullable, why is the value 0/null? I probably just don't understand our tags.